ci_framework/playbooks/ceph.yml

```diff
@@ -180,6 +180,14 @@
         name: cifmw_cephadm
         tasks_from: pools
 
+    - name: Deploy additional daemons
+      import_role:
+        name: cifmw_cephadm
+        tasks_from: rgw
+      vars:
+        cifmw_cephadm_rgw_network: 192.168.122.0/24
```

Comment:
@fultonj  I see this subnet range is hardcoded in other parts of the `ceph` playbook, so I think it's ok for now (and customized via `-e @extra_vars.yaml` ?

Reply:
@fmount that range is the "default" one in libvirt and CRC - at least "old" CRC. Hopefully everything is done in the framework to allow updating it easily in case of need - but install_yamls itself has a lot of hardcoded occurrences.

Reply:
ack I see, this means that for now we should keep the variable with that hardcoded value, and hopefully later in time we can feed the Ceph related network variables w/ something gathered from the infra deployment.